receipt/SaleReceipt_fr_CA-QC.tpl

```diff
@@ -227,10 +231,6 @@ table div.line_description_item_fee {
 	padding-left: 10px;
 }
 
-.footerSectionTitle + table div.line_description_item_fee {
-	padding-left: 25px;
-}
-
```

Comment:
question: It seems this rule is declared on all receipt templates. Why are we removing it here?

Reply:
Not sure what happened here as to mark it for delete. It should now be updated